pkgs/applications/audio/zynaddsubfx/system-libuv.patch

```diff
@@ -0,0 +1,113 @@
+diff --git a/Makefile b/Makefile
+index f3e3be2..2398852 100644
+--- a/Makefile
++++ b/Makefile
+@@ -1,8 +1,3 @@
+-UV_DIR    = libuv-v1.9.1
+-UV_FILE   = $(UV_DIR).tar.gz
+-UV_URL    = http://dist.libuv.org/dist/v1.9.1/$(UV_FILE)
+-	 
+-
+ all:
+ 	ruby ./rebuild-fcache.rb
+ 	cd deps/nanovg/src   && $(CC) nanovg.c -c -fPIC
+@@ -10,12 +5,12 @@ all:
+ #	cd deps/pugl         && python2 ./waf configure --no-cairo --static
+ 	cd deps/pugl         && python2 ./waf configure --no-cairo --static --debug
+ 	cd deps/pugl         && python2 ./waf
+-	cd src/osc-bridge    && CFLAGS="-I ../../deps/$(UV_DIR)/include " make lib
++	cd src/osc-bridge    && make lib
```

Comment:
We should not call make directly like this because it does not adhere to $NIX_BUILD_CORES

Reply:
This is a patch on the root Makefile, so after refactoring the `installPhase` like you suggested, any environment variables passed to this Makefile will be propagated to the child process.

Reply:
Actually, I don't think that arguments like `--parallel` will be propagated, but I'm sure there are other projects in nixpkgs that do this, and it probably doesn't make sense to have nested parallel builds.